src/material/chips/chip.scss

```diff
@@ -695,16 +695,20 @@ $_avatar-trailing-padding: 8px;
 
   // Used as a state overlay.
   &::after {
-    $offset: 2px;
+    $_touch-target-size: 48px;
+    $_ripple-size: 24px;
     content: '';
     display: block;
     opacity: 0;
     position: absolute;
-    top: 0 - $offset;
-    bottom: 0 - $offset;
-    left: 8px - $offset;
-    right: 8px - $offset;
-    border-radius: 50%;
+    // Increases touch target size to improve accessibility and fix b/286959517
+    z-index: 100;
+    pointer-events: all;
+    box-sizing: border-box;
+    padding: calc(($_touch-target-size - $_ripple-size)/2);
+    margin: calc((($_touch-target-size - $_ripple-size)/2) * -1);
+    -webkit-background-clip: content-box;
```

Comment:
background-clip: content-box has good browser support, this shouldn't be needed. If you added it because of some lint check you can supress the lint. https://caniuse.com/mdn-css_properties_background-clip_content-box

Reply:
Updated based on your recommendations and confirmed appropriate touch-target size: https://screenshot.googleplex.com/4wVGKZ6nV3s7cy6.